migrations/2_deploy_contracts.js

```diff
@@ -1,9 +1,11 @@
 var RelayHub = artifacts.require('./RelayHub.sol')
 var StakeManager = artifacts.require('./StakeManager.sol')
 var Penalizer = artifacts.require('./Penalizer.sol')
+var SampleRecipient = artifacts.require('./test/TestRecipient.sol')
 
 module.exports = async function (deployer) {
   await deployer.deploy(StakeManager)
   await deployer.deploy(Penalizer)
   await deployer.deploy(RelayHub, 16, StakeManager.address, Penalizer.address)
+    deployer.deploy(SampleRecipient)
```

Comment:
We should either use migration as they should (for target/production contract: the hub and friends), or don't use them at all. 
We shouldn't "migrate" test code.

Reply:
truffle is a test environment first and foremost. I don't see any issue with having it here. We don't use it yet to deploy production code.